### DISCUSSION TITLE
New Model Refact Code 1.6B for code

### DISCUSSION THREAD

**Discussion by forrackun** [ggml-org/ggml#503] [General]
https://refact.ai/blog/2023/introducing-refact-code-llm/

**Comment by Green-Sky** [comment]
their previous blog post is a good explanation for their choices (alibi, lion, gqa...)
https://refact.ai/blog/2023/applying-recent-innovations-to-train-model/

**Comment by Jipok** [comment]
https://github.com/ggerganov/llama.cpp/discussions/3013

**Comment by Green-Sky** [comment]
tracking issue: https://github.com/ggerganov/llama.cpp/issues/3061
